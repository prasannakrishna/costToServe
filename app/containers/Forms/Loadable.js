/**
 *
 * Asynchronously loads the component for Sample2Page
 *
 */

import loadable from 'loadable-components';

export default loadable(() => import('./index'));
