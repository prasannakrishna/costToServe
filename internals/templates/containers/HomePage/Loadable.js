/**
 * Copyright Â© 2018, JDA Software Group, Inc. ALL RIGHTS RESERVED.
 * <p>
 * This software is the confidential information of JDA Software, Inc., and is licensed
 * as restricted rights software. The use,reproduction, or disclosure of this software
 * is subject to restrictions set forth in your license agreement with JDA.
 */
/**
 * Asynchronously loads the component for HomePage
 */
import Loadable from 'react-loadable';

export default Loadable({
  loader: () => import('./index'),
  loading: () => null,
});
