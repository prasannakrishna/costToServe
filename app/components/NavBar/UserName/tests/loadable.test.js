/**
 * Copyright Â© 2018, JDA Software Group, Inc. ALL RIGHTS RESERVED.
 * <p>
 * This software is the confidential information of JDA Software, Inc., and is licensed
 * as restricted rights software. The use,reproduction, or disclosure of this software
 * is subject to restrictions set forth in your license agreement with JDA.
 */
/* eslint-disable linebreak-style */
import React from 'react';
import renderer from 'react-test-renderer';

import Loadable from '../Loadable';

describe('<Loadable />', () => {
  it('should render correctly', () => {
    const comp = renderer.create(
      <Loadable />
      );
    expect(comp.toJSON()).toMatchSnapshot();
  });
});
