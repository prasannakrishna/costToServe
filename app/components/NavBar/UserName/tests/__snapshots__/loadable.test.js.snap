// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loadable /> should render correctly 1`] = `null`;
